Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2522,7 +2522,7 @@ HRESULT Stage(CGContextRef context, ID2D1DeviceContext* deviceContext) override
             deviceContext->Clear({ 0, 0, 0, 0 }); // Clear the original target to transparent black.
             RETURN_IF_FAILED(inputBitmap.As(&copiedImage));
         }
-        return __super::Stage(context, deviceContext);
+        return __super ::Stage(context, deviceContext);
```

Comment:
LOL it's just that i need to disable and re-enable it every darn time.
I'll do a regex replacement at the end.

---
In reply to: [107502011](https://github.com/Microsoft/WinObjC/pull/2249#discussion_r107502011) [](ancestors = 107502011)